upcoming-release-notes/1099.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Bugfix
+authors: [Shazib]
+---
+
+Fix electron app issues
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you please be a bit more specific of what issues are fixed here?